plugins/SitesManager/templates/_siteWithoutDataTabs.twig

```diff
@@ -192,6 +195,12 @@
         </div>
     {% endif %}
 
+    {% if showGAImportTab %}
+        <div id="ga-import" class="col s12">
+            {{ postEvent('Template.embedGAImportNoData', ga3Used) }}
```

Comment:
@bx80 The code to show the tab and the event to load the below UI is in this PR and the reason we see Cloudflare currently is due to we are setting GA and Cloudflare in tests and the code to show GA tab over Cloudflare is in `getActiveTabOnLoad` which is again in this PR
![Screenshot from 2023-07-12 06-41-23](https://github.com/matomo-org/matomo/assets/14902277/c2c9e2e8-4654-4867-9e63-0878910f5740)